### ISSUE TITLE
Automatically resize ataraxis mode on VimResize

### ISSUE THREAD

**Issue by henriquehbr** [Pocco81/true-zen.nvim#53]
Would it be interesting to implement a `VimResize` autocommand which adjusts the middle buffer dimensions whenever the window gets resized?

The actual behavior is whenever a window gets resized, the middle buffer isn't adjusted, it's needed to run `:TZAtaraxis` twice in order to toggle off and then on again for centering it accordingly to the current terminal window boundaries

**Comment by Pocco81** [comment]
Good idea, I'll add it! But I don't think I'll make it to "run TZAtaraxisOff and then TZAtaraxisOn when nv was resized".

I did test that but I ran into a lot of problems. For starters, nvim got stock in a infinit loop, because when I turned Ataraxis off it meant that nv was resized, which triggered the auto command and it went like that... Also, I use tmux and have the tmux integration enabled. Which means that untoggling the whole thing will just... Look messy.

Maybe something like a "denounce delay" would do the trick to avoid getting stuck in an infinite loop?
